types/key/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for key 0.1
+// Project: https://github.com/adlawson/key
+// Definitions by: hayke102 <https://github.com/hayke102>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Code {
+    special: Special;
+    arrow: Arrow;
+    punctuation: Punctuation;
+    alnum: Alnum;
+}
+
+declare const key: {
+    code: Code,
+    // brand: Brand;
+
+    get(keyCode: number): Reference,
+
+    is(ref: Reference, keyCode: number): boolean;
+}
+export = key;
```

Comment:
If you also want it to be accessible in the browser, I believe you'll need to add a `declare global` (e.g. see types/anchor-js).